building/build-debs/homeworld-admin-tools/src/iso.py

```diff
@@ -69,9 +70,12 @@ def gen_iso(iso_image, authorized_key, cdpack=None):
         preseeded = resource.get_resource("preseed.cfg.in")
         generated_password = util.pwgen(20)
         creation_time = datetime.datetime.now().isoformat()
+        git_hash = get_resource("GIT_VERSION").decode().rstrip().encode()
```

Comment:
It would probably be worth adding a function to `version.py` that gets the current version, because we don't want to duplicate that logic.